cp2k_output_tools/blocks/scf.py

```diff
@@ -71,7 +71,7 @@
 
 
 INNER_SCF_START_RE = re.compile(r"^\s+ SCF\ WAVEFUNCTION\ OPTIMIZATION", re.VERBOSE | re.MULTILINE)
-INNER_SCF_END_RE = re.compile(
+INNER_SCF_CONV_RE = re.compile(
```

Comment:
Please note that the blocks themselves can be used by other tools, changing the names will break them.
But if you think there is good reason to do so, please continue. The only thing I would then ask you is whether you can add a new entry in `CHANGELOG.md` for the next minor version (major.minor.patch) to make sure we don't forget to bump it on the next release for the backwards incompatible change.
The reason I called it `SCF_END_RE` is because it's notoriously difficult to reliably determine whether the CP2K (SCF) finished properly. The idea was to assume that the presence of that line indicates a finished SCF, everything else not.

Reply:
Sorry, I didn't take that into consideration. I just thought that since the energies are parsed further below in the output file and SCF_END_RE would refer to the end of the inner SCF cycle it would make sense to rename it.

I believe in this having backwards compatibility is more important, so reversed this change.